docs/package/gluon-scheduled-domain-switch.rst

```diff
@@ -31,8 +33,11 @@ Example::
     target_domain = 'new_domain',
     switch_after_offline_mins = 120,
     switch_time = 1546344000, -- 01.01.2019 - 12:00 UTC
-    connection_check_targets = {
+  },
+
+  connection_check = {
```

Comment:
As you change the name of the site.conf field here, can you please change it to check_connection instead?
I assume that the package was named this way to allow a unified naming for check packages later. E.g. check-throughput, check-wan-connection, check-batman-iv-tq or whatever. Thus it would be nice if the config options start with check, too.

Reply:
good catch, the same order of those words should be used in the whole package @rubo77